persistence-modules/spring-data-jpa-filtering/src/main/java/com/baeldung/projection/view/PersonAddressDTO.java

```diff
@@ -0,0 +1,21 @@
+package com.baeldung.projection.view;
+
+public class PersonAddressDTO {
+
+    private final String firstName;
```

Comment:
Ditch this `final` or make both `final`. I prefer to avoid `final` unless I'm very specifically making an immutable object.